src/app/services/query.service.ts

```diff
@@ -60,28 +60,31 @@ export class QueryService {
     }
   }
 
-  downloadCsv(indices, fields, sort, project) {
+  downloadCsv(indices, fields, sort, project, fileFormat, accession) {
```

Comment:
Could we rename this function to have a more general name like "downloadFile" as it is now used to download different file formats?